src/components/ServiceAreasDrawer/ServiceAreasDrawer.tsx

```diff
@@ -80,7 +80,7 @@ let parse = parseRows(COLUMNS, (([county, zip], rowIndex) => {
   // validate that counties exist
   let badCounty = pairs.value().find(([county]) => !(county in COUNTIES_TO_ZIPS))
   if (badCounty) {
-    return new ParseError(rowIndex, 0, COLUMNS[0], `County "${badCounty[0]}" is not supported`)
+    return new ParseError(rowIndex, 0, COLUMNS[0], `We don't support county "${badCounty[0]}" yet. Reach out to us at hello@bayesimpact.org.`)
```

Comment:
More friendly error messages

Reply:
@viviandien I like this idea. should we create an email address for support / question? Esp. if you will be demoing it soon and keeping it available online. hello is probably not the best email though, thoughts?